projects/components/src/quick-search/quick-search.component.spec.ts

```diff
@@ -855,6 +855,30 @@ describe('QuickSearchComponent', () => {
             expect(this.quickSearch.getSearchResultSectionTitles().length()).toEqual(1);
             expect(this.quickSearch.getSearchResultItems().text()).toEqual('copy');
         });
+
+        it('can set filter values through the QuickSearchService', function (this: Test): void {
```

Comment:
I don't see a need to mention quick search service as part of this test. The QuickSearch Service isn't doing much...

I do think it should be doing the filtering. We could move a lot of code to the service, simplifying the UI, its tests, and this functionality can be tested without the need for a UI.

Please create a task sharing your thoughts on how we can move all the data processing to the quick service.

Reply:
https://jira.eng.vmware.com/browse/VDUCC-694